src/services/__tests__/web-monitoring-api.test.js

```diff
@@ -1,87 +0,0 @@
-/* eslint-env jest */
-
-import WebMonitoringApi from '../web-monitoring-api';
-import fetch from 'fetch-mock';
-
-describe('WebMonitoringApi', () => {
```

Comment:
We *should* have tests for the remaining parts of this module, so I think it would be good to leave the file and include some `test.todo()` calls, e.g:

```js
describe('WebMonitoringApi', () => {
  test.todo('addChangeToDictionary() updates the dictionary spreadsheet');

  test.todo('addChangeToImportant() updates the important spreadsheet');
});
```

Reply:
Makes sense! I could just write a couple of tests and add that to this PR. Is that ok? Or would you prefer a separate one?

Reply:
I tend towards keeping it separate so any back-and-forth about them doesn’t hold up this PR, but if you feel confident whipping up some quick tests, go for it. 👍

Reply:
Ok, I can add them for now and if it seems like they need to be refactored a bunch, I can pull them out.